pkgs/applications/window-managers/i3/lock-fancy.nix

```diff
@@ -3,13 +3,13 @@
 }:
 
 stdenv.mkDerivation rec {
-  rev = "b7196aaff72b90bb6ea0464a9f7b37d140db3230";
-  name = "i3lock-fancy-2016-05-05_rev${builtins.substring 0 7 rev}";
+  rev = "546ce2e71bd2339f2134904c7d22062e86105b46";
+  name = "i3lock-fancy-2016-10-13_rev${builtins.substring 0 7 rev}";
```

Comment:
do we want to adapt to the new scheme, which includes the `-unstable-` infix here as well?